adr/0011-sap-system-database-aggregate-split.md

```diff
@@ -0,0 +1,33 @@
+# 11. SAP system and database aggregate split
+
+Date: 2024-04-04
+
+## Status
+
+Accepted
+
+## Context
+
+In the initial implementation of Trento, the SAP system and database states are stored in the same `sap_system.ex` aggregate. This is making the assumption that the relationship between a SAP system and a database is of 1 to 1, meaning that a database is only used by one SAP system. This is not an accurate description of the reality, as a database can handle multiple SAP systems in a multi-tenant environment.
+
+In this scenario, when a multi-tenant database is discovered, it only can manage a unique SAP system and its application instances, making the multi-tenant scenario not supported.
+
+This must be reverted, as a multi-tenant scenario is pretty common, and many deployments have them.
```

Comment:
We're not really reverting anything, since we don't have a previous state of things. :)

```suggestion
This limitation must be overcome, as a multi-tenant scenario is pretty common, and many deployments have them.
```